sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -776,7 +776,7 @@ ErrorReport buildErrorReport(ManagedErrorLog log) {
                 if (file.length() > 0) {
                     try {
                         throwable = FileManager.readObject(file);
-                    } catch (IOException | ClassNotFoundException | StackOverflowError e) {
+                    } catch (java.lang.Exception | StackOverflowError e) {
```

Comment:
```suggestion
                    } catch (IOException | RuntimeException | StackOverflowError e) {
```

The only checked exception is IOException so that means unchecked exceptions can only be RuntimeException or a sub-class.